src/main/scala/edu/berkeley/ce/rockslicing/BlockVTK.scala

```diff
@@ -0,0 +1,213 @@
+package edu.berkeley.ce.rockslicing
+
+import breeze.linalg
+import breeze.linalg.{DenseVector, DenseMatrix}
+
+object BlockVTK {
+
+  /**
+    * Calculates the rotation matrix to rotate the input plane (specified by its normal)
+    * to the desired orientation (specified by the desired normal)
+    * @param n_current: Current normal of plane
+    * @param n_desired: Desired new normal
+    * @return 3*3 rotation matrix
+    */
+  private def rotationMatrix(n_current: (Double, Double, Double), n_desired: (Double, Double, Double)):
```

Comment:
So this is the same code as in `Block` right? Why not put this as a public method in the `Block` object (not class), and then we can use it as a utility. That way, we reuse code and only have to make changes in one place if it needs updating.